tests/ciaopeople-it.spec.ts

```diff
@@ -0,0 +1,8 @@
+import generateCMPTests from "../playwright/runner";
+
+generateCMPTests('ciaopeople-it', [
```

Comment:
The name in tests should match the one in the rule
```suggestion
generateCMPTests('ciaopeople.it', [
```